code/game/gamemodes/game_mode.dm

```diff
@@ -325,7 +325,7 @@
 		p_rep = SSpersistence.antag_rep[p_ckey]
 		p_rep = p_rep == null ? 0 : p_rep
 

```

Comment:
This:
`current += min(p_rep + DEFAULT_ANTAG_TICKETS, MAX_TICKETS_PER_ROLL)`
should be moved up here before current is compared, shouldn't it?

Otherwise a current candidate's chances are based on the tickets of the candidate before them, instead of their own.

Current Behavior:
```
current = 1
antag_select = 180
-------------------------------
candidate 1's tickets = 200
antag_select (180) <= current (1)  // candidate 1 loses
current += candidate 1's tickets (200)     //current is 201, the next candidate wins.

candidate 2's tickets = 100
antag_select (180) <= current (201) //candidate 2 wins, but candidate 1 should've
```